consumer/migration/hermes_migrator.go

```diff
@@ -80,43 +89,31 @@ type HermesClient interface {
 // Start begins migration from old hermes to new
 func (m *HermesMigrator) Start(id string) error {
 	chainID := config.GetInt64(config.FlagChainID)
-	if !m.st.isMigrationRequired(chainID, id) {
-		log.Info().Msg("Migration is already done")
-		return nil
-	}
-
-	registered, err := m.isRegistered(chainID, id)
```

Comment:
We still want to check this as if you are not registered, migration is not required.